frontend/src/styles/globals.css

```diff
@@ -13,8 +13,10 @@
   --color-gray: #808080;
   --color-light-gray: #bfbfbf;
   --color-white: #fff;
+  --color-red: #f73f52;
   --font-family-body: "Work Sans", sans-serif;
   --font-family-title: "Spicy Rice", cursive;
+  --font-size-xxs: 12px;
```

Comment:
We aren't using this for `font-size` but padding and margins

Reply:
@mirhamasala this is a bit out of scope for this PR, we will rename this constants to better fit usage if you agree

Reply:
I agree, and in that case, we should just use `12px` directly and remove this var.

Reply:
Or we can leave it as I suggest Ricardo and then simply rename these variables across the project. Seems less work than renaming them and then going to each individual component to start using them, or am I missing something?

Reply:
I don't see how using `12px` directly isn't better? This is incorrect.

Reply:
I misunderstood. I meant adding `spacing` variables, not renaming the existing `font-size` ones.